hera_mc/cm_hookup.py

```diff
@@ -7,15 +7,14 @@
 This is computes and displays part hookups.
 
 """
-from __future__ import absolute_import, division, print_function
+from __future__ import print_function
 from tabulate import tabulate
```

Comment:
Why were these removed? They enable behavior that is both more intuitive *and* consistent with Python 3.